DashKit/Mocks/MockPodCommManager.swift

```diff
@@ -56,6 +56,17 @@ public class MockPodCommManager: PodCommManagerProtocol {
     
     public var bleConnected: Bool = true
 
+    public func update(for state: DashPumpManagerState) {
+        guard state.suspendState != nil else {
+            setDeactivatedState()
+            return
+        }
+        
+        if let reservoirUnitsRemaining = state.reservoirLevel?.rawValue {
+            podStatus.reservoirUnitsRemaining = reservoirUnitsRemaining
```

Comment:
[nit] If `state.reservoirLevel == nil` should `podStatus. reservoirUnitsRemaining` be set to `nil`, too? So,
```
podStatus.reservoirUnitsRemaining = state.reservoirLevel?.rawValue
```

Reply:
This uses SDK protocol. I'll update to:

`podStatus.reservoirUnitsRemaining = state.reservoirLevel?.rawValue ?? 0`